graphql-dgs-spring-webmvc-autoconfigure/build.gradle.kts

```diff
@@ -14,9 +14,14 @@
  * limitations under the License.
  */
 
+plugins {
+    id("org.jetbrains.kotlin.plugin.allopen") version Versions.KOTLIN_VERSION
+}
+
 dependencies {
     api(project(":graphql-dgs"))
     api(project(":graphql-dgs-spring-webmvc"))
+    api("org.hibernate.validator:hibernate-validator")
```

Comment:
can it be `implementation`?

Reply:
It could, but if apps did not bring it in then it would bypass validation.